code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -16,7 +16,7 @@
 	var/convert_damage = FALSE //If you want to convert the caster's health to the shift, and vice versa.
 	var/convert_damage_type = BRUTE //Since simplemobs don't have advanced damagetypes, what to convert damage back into.
 
-	var/shapeshift_type
+	var/mob/living/shapeshift_type
```

Comment:
This variable is still a type. You need to do initial() if you want to access variables.